apps/marketplace/components/BuyerTraining/BuyerTrainingIntroductionStage.js

```diff
@@ -22,7 +22,11 @@ const BuyerTrainingIntroductionStage = props => (
     <ul>
       <li>
         The{' '}
-        <a href="/api/2/r/learning-design-standards" rel="noopener noreferrer" target="_blank">
+        <a
```

Comment:
This link can be updated in the database.  See if you can work out where it's updated.